setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+import digits
+
+setup(
+    name = "digits",
+    packages = find_packages(),
```

Comment:
We need to exclude some directories here. `pip install .` is freezing on me because it's opening, reading (and possibly zipping?) everything under `digits/jobs/` and `examples/object-detection/kitti-data/`, for example.